CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to [bpmn-js](https://github.com/bpmn-io/bpmn-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+* `DEPS`: update to `diagram-js@14.7.0`
```

Comment:
Let's keep our existing order `FEAT` -> `FIX` -> `CHORE` -> `DEPS`

Reply:
Let's also make this more descriptive: What problem do we solve? We now allow coloring of markers via CSS.

Reply:
We could add https://github.com/bpmn-io/bpmn-js/pull/2173/files#r1617203732 as an additional `FIX` entry in the changelog.

Reply:
I added the descriptive message as a `FIX` and the technical description as a `CHORE` (how we render SVG is not that relevant to the user): [88e70a5](https://github.com/bpmn-io/bpmn-js/pull/2173/commits/88e70a5ab8c07ea45dab33f380fc78be0671dc29)

Reply:
:+1: